training_curve.py

```diff
@@ -0,0 +1,250 @@
+"""
```

Comment:
can we move this file to the oneoffs/ directory?

Reply:
Done